wiki/en/en-Software-Synth.md

```diff
@@ -29,14 +29,14 @@ For ALSA2JACK MIDI (`a2jmidid`) connection it is necessary to install the a2j pa
 
 ### Connecting ALSA, JACK and MIDI for a Synth for Jamulus
 Now we look an the workflow to use the whole infrastructure in Jamulus.
-* **(Jamulus Installation and JACK)** Use the [install script](https://github.com/corrados/jamulus/wiki/Linux-Client-Install-Script) for Jamulus on Linux or [install Jamulus with the manual in the wiki](https://github.com/corrados/jamulus/wiki/Installation-for-Linux). With this installation we have JACK and QJackCtl installed for Jamulus. 
```

Comment:
Using atom text editor?

Reply:
VScode usualy, but Atom here in case I have to edit the index (VSCode re-indent)
Why this was deleted?

Reply:
Atom removes spaces as far as I saw multiple times in my git history. It's ok, but a bit strange...

Reply:
@ann0see (As I understand you use Atom?)
> Atom removes spaces as far as I saw multiple times in my git history. It's ok, but a bit strange...

FYI: https://stackoverflow.com/questions/30403150/atom-text-editor-remove-trailing-whitespace-on-save

- Settings > Packages
- Search for "whitespace" (should be enabled) > Settings
- Uncheck "Remove Trailing White Space"

That should fix this behavior. ("Keep Markdow Line Break Whitespace" must be checked if not)

HTH.